.github/workflows/go.yml

```
@@ -0,0 +1,27 @@
+name: Go
+on: [push, pull_request]
+#  push:
+#    branches: [ main ]
+#    paths: '**.go'
+#  pull_request:
+#    branches: [ main ]
+#    paths: '**.go'
+jobs:
+  Go:
+    name: lint
+    runs-on: ubuntu-latest
+    steps:
+      - run: go list -e -json -compiled=true -test=true -export=true -deps=true -find=false -- ./...
+      - uses: actions/checkout@v2
+      - name: golangci-lint
+        uses: golangci/golangci-lint-action@v2
+        with:
+          version: latest
+          
+          args: ./...
+          # Optional: working directory, useful for monorepos
+          # working-directory: algorithms/Go
+          
+          # Optional: show only new issues if it's a pull request. The default value is `false`.
+          # only-new-issues: true
+      - run: go test $(go list ./... | grep -v /sbom | grep -v /rsa)
```

```suggestion
      - run: go test $(go list ./...)
```
This repo does not have directories `/sbom` and `/rsa`.